docs/qe-available-tasks/sealights/go-instrumentation/go-instrumentation.md

```diff
@@ -0,0 +1,126 @@
+# Sealights Go Instrumentation Tekton Task
+
+This Tekton task automates the process of instrumenting Go code with Sealights for quality analytics and testing. It retrieves source code from a trusted artifact, instruments it with Sealights, and creates a new trusted artifact with the instrumented code. This task supports Pull Request and Push events and provides flexibility to exclude specific Go packages from instrumentation.
+
+## Overview
+
+This task performs the following steps:
+
+1. **Retrieves** the source code from a trusted artifact.
+2. **Instruments** the Go code using Sealights.
+3. **Creates** a new trusted artifact containing the instrumented code.
+
+The task can be triggered by different events (e.g., Pull Request, Push) and allows users to exclude specific Go packages from the instrumentation process.
+
+## Parameters
+
+| Name                  | Type     | Default       | Description                                                                                   |
+|-----------------------|----------|---------------|-----------------------------------------------------------------------------------------------|
+| `source-artifact`     | `string` | -             | The Trusted Artifact URI pointing to the source code.                                         |
+| `go-version`          | `string` | -             | The Go version to use (e.g., `1.21.3`).                                                       |
```

Comment:
would be maybe worth to document that the supported versions of Go are based on what's available in the [catalog](https://catalog.redhat.com/software/containers/ubi8/go-toolset/5ce8713aac3db925c03774d1)?

Reply:
good point!